source/extensions/omni.isaac.lab/omni/isaac/lab/sim/simulation_cfg.py

```diff
@@ -175,6 +175,9 @@ class SimulationCfg:
     render_interval: int = 1
     """The number of physics simulation steps per rendering step. Default is 1."""
 
+    antialiasing_mode: str = "DLSS"
+    """The antialiasing mode to use for rendering. Default is DLSS."""
```

Comment:
I feel soon we reach a situation where we may just want to expose a lot of the renderer settings through the simulationcfg settings (similar to physics settings).

Should we already take a small step in that direction and make `render` attribute with `RenderCfg`?